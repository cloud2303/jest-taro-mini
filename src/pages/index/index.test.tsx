
import {render} from '@testing-library/react'
import Index from "./index";
//jest --updateSnapshot 更新快照
//最好用watch模式，w u交互更新
describe("Index", () => {
  it("should render", () => {
    const {container} = render(<Index />)
    
    expect(container).toMatchSnapshot();
    const title = container.querySelector(".title")
    expect(title).toBeInTheDocument()
    expect(title).toHaveTextContent("Hello world!")
  });
})