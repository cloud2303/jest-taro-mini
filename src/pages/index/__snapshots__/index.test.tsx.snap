// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render 1`] = `
<div>
  <taro-view-core
    class="index"
  >
    <taro-text-core
      class="title"
    >
      Hello world!ccc
    </taro-text-core>
  </taro-view-core>
</div>
`;
